shop/src/main/java/ro/msg/learning/configuration/MapperConfig.java

```diff
@@ -0,0 +1,20 @@
+package ro.msg.learning.configuration;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+
+@Configuration
+public class MapperConfig {
+	@Bean("modelMapper")
+	public ModelMapper modelMapper() {
+		return new ModelMapper();
+	}
+
+	@Bean("csvMapper")
```

Comment:
Types are different why need a qualifier?

Reply:
Because maybe CSVMapper is not annotated with @Component